kernel/content/ezcontentoperationcollection.php

```diff
@@ -866,6 +870,11 @@ static public function removeNodes( array $removeNodeIdList )
 
             if ( !isset( $objectIdList[$objectId] ) )
                 $objectIdList[$objectId] = eZContentObject::fetch( $objectId );
+            eZAudit::writeAudit( 'location-remove', array( 'Parent Node ID' => $node->attribute( 'parent_node_id' ),
+                                              'Node ID' => $nodeId,
```

Comment:
Can we be clear about which node ID was removed, and be clear on what is the (new) main node ID?